docs/aws/Index.rst

```diff
@@ -1,9 +1,9 @@
 .. _index_aws:
 
-Asset Tracker Cloud Example for AWS
-###################################
+AWS cloud components 
+#####################
 
-The :ref:`Asset Tracker Cloud Example <project>` for AWS is a reference implementation of a serverless backend for an IoT product developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_.
+The AWS cloud components of the :ref:`nRF Asset Tracker <project>` are a reference implementation of a serverless backend for an IoT product developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_.
```

Comment:
```suggestion
The nRF Asset Tracker provides a reference implementation of a serverless backend for an IoT product via the AWS cloud components.
It is developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_. 
```

Reply:
There is no such thing as "AWS cloud components".
AWS is a standalone term, and means "Amazon Web Services".
```suggestion
The nRF Asset Tracker provides a reference implementation of a serverless backend for an IoT product using AWS.
It is developed using `AWS CDK <https://aws.amazon.com/cdk/>`_ in `TypeScript <https://www.typescriptlang.org/>`_. 
```